aiida_vasp/parsers/base.py

```diff
@@ -126,7 +128,7 @@ def _get_handler(self, name):
                 # For the permanent content to be parsed we can use the fact that
                 # self.retrieved is a FolderData datatype in AiiDA.
                 try:
-                    with self.retrieved.open(name) as handler:
+                    with self.retrieved.open(name, open_mode) as handler:
```

Comment:
So it may be better calling with the keyword `mode`:
```
self.retrieved.open(name, open_mode)
```
->
```
self.retrieved.open(name, mode=open_mode)
```

Reply:
Very good point. I will change all wording to `mode` as that is the standard.